com.unity.ml-agents/Runtime/Inference/TensorApplier.cs

```diff
@@ -52,16 +52,25 @@ public interface IApplier
             Dictionary<int, List<float>> memories,
             object barracudaModel = null)
         {
-            actionSpec.CheckNotHybrid();
-
+            bool useDeprecated = false;
+            if (barracudaModel != null)
```

Comment:
Can we just early out if the model is null? Might make some of the code cleaner. (same for TensorGenerator below)

Reply:
done